unity-renderer/Assets/Scripts/MainScripts/DCL/Configuration/Configuration.cs

```diff
@@ -236,4 +236,9 @@ public static class PhysicsLayers
         public static int playerInfoCardMenu = LayerMask.NameToLayer("PlayerInfoCardMenu");
         public static int avatarTriggerMask = LayerMask.GetMask("AvatarTriggerDetection");
     }
+
+    public static class APIUrls
+    {
+        public const string AVATAR_TEXTURE_API_URL = "https://synapse.decentraland.org/profile-pictures/";
```

Comment:
As a rule of thumb we rarely compose urls ourselves in the renderer. Shouldnt kernel provide this with the correct environment?